examples/twittermap/web/public/javascripts/timeseries/controllers.js

```diff
@@ -95,6 +95,7 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
         controller: 'TimeSeriesCtrl',
         link: function ($scope, $element, $attrs) {
           var chart = d3.select($element[0]);
+          console.log(d3);

```

Comment:
Remove, if not using anymore

Reply:
Might have left it when I was debugging the time-series issue